src/coreclr/debug/ee/controller.cpp

```diff
@@ -25,6 +25,8 @@ const char *GetTType( TraceType tt);
 
 #define IsSingleStep(exception) ((exception) == EXCEPTION_SINGLE_STEP)
 
+#define TailCallThatReturns 1
+#define StoreTailCallArgs 2
```

Comment:
nit: I suggest making these enum's and adding them to controller.h for consistency with the rest of the controller code.  for example:

```c++
enum TAILCALL_TYPE
{
    TAILCALL_ RETURNS               = 0x1
    TAILCALL_STORE_CALL_ARGS = 0x2
};
```